export/securedrop_export/archive.py

```diff
@@ -26,7 +26,6 @@ class Metadata(object):
     """
 
     METADATA_FILE = "metadata.json"
-    SUPPORTED_ENCRYPTION_METHODS = ["luks"]
```

Comment:
For Git posterity, can you explain why in the commit message?  (Or squash with c6f871c486bb68089cd23fa7cd3f62ad42a26e42.)